migrations/1629229061520-create-collection-index.js

```diff
@@ -0,0 +1,108 @@
+const path = require('path');
+const { logger: rootLogger, mongo } = require('../app/helpers');
+
+module.exports.up = async () => {
+  const logger = rootLogger.child({
+    gitHash: process.env.GIT_HASH || 'unspecified',
+    migration: path.basename(__filename)
+  });
+
+  const database = await mongo.connect(logger);
+
+  logger.info('Start migration');
+
+  let collection;
```

Comment:
Using variables from outside `Promise` causing incorrect indexes mapping to collections.
For example, I see the `trailing-trade-symbols-key-idx` added in the **trailing-trade-common** collection.
I don't know if that was the main reason but when I move the variables inside the `Promise`, it solves the issue.

Reply:
Oh really? Interesting. Let me test it out.

Reply:
You are right! Gee, how did I miss this. Let me fix it up.